packages/smart-wallet/src/smartWallet.js

```diff
@@ -140,8 +138,8 @@ const { StorageNodeShape } = makeTypeGuards(M);
  *   offerToPublicSubscriberPaths: MapStore<string, Record<string, string>>,
  *   offerToUsedInvitation: MapStore<string, Amount>,
  *   purseBalances: MapStore<RemotePurse, Amount>,
- *   updatePublishKit: PublishKit<UpdateRecord>,
- *   currentPublishKit: PublishKit<CurrentWalletRecord>,
+ *   updateRecorderKit: import('@agoric/zoe/src/contractSupport/recorder.js').RecorderKit<UpdateRecord>,
+ *   currentRecorderKit: import('@agoric/zoe/src/contractSupport/recorder.js').RecorderKit<CurrentWalletRecord>,
```

Comment:
recorder.js is a zoe concern? hm.

(I see later your comment about an `@agoric/contracts` package. makes sense)